src/kirby/components/chip/chip.component.tns.html

```diff
@@ -0,0 +1,3 @@
+<StackLayout class="chip" [class.selected]="isSelected">
```

Comment:
Keep naming on sync:
```suggestion
<StackLayout class="chip" [class.is-selected]="isSelected">
```

Reply:
Done